packages/storybook/src/docs/component-status.stories.ts

```diff
@@ -559,11 +560,15 @@ const metadata: Meta<TableArgs> = {
     },
     // prettier-ignore
     render: createUserSelectedThemeStory(html<TableArgs>`
+        <style class="code-hide">
+            ${tableTag} {
+                height: var(${tableFitRowsHeight.cssCustomProperty});
+                max-height: unset;
```

Comment:
unset isn't likely to be what you need: https://developer.mozilla.org/en-US/docs/Web/CSS/unset
The initial value of the non-inheritable max-height property is none: https://developer.mozilla.org/en-US/docs/Web/CSS/max-height#formal_definition

We should just set it to none if that's the intended usage